src/message/messageActionSheet.js

```diff
@@ -39,6 +45,13 @@ const reply = ({ message, dispatch, auth }) => {
 };
 reply.title = 'Reply';
 
+const replyWithMention = ({ message, dispatch, auth }) => {
+  const narrow = getNarrowFromMessage(message, auth.email);
+  dispatch(doNarrow(narrow, message.id));
+  dispatch(draftUpdate(narrow, `@**${message.sender_full_name}** `));
+};
+replyWithMention.title = 'Reply @%user%';
```

Comment:
How does `Reply to @user` sound here?

Reply:
Sounds way better than without the 'to' :)
But I don't think we need the '@' in either case.

Reply:
Indeed, it sound better. Thanks!